spec/mailerlite/clients/campaigns_spec.rb

```diff
@@ -78,7 +78,7 @@
     end
 
     it 'has correct response' do
-      expect(response).to eq({})
+      expect(response).to be_nil
```

Comment:
hmm, what changed the response? some middlewares now returning response differently?

Reply:
I think so. JSON parsing has been pulled into Faraday’s core from the old middleware gem so something must have changed there.

My initial thought was that responses to DELETE requests always return 204 No Content so we could leave it as nil and have a breaking change. However looking at MDN’s docs about DELETE it’s entirely valid to include a response body so it’s probably better to change these delete methods to default to empty hash instead. I can’t think of a good reason to change it to nil just because the middleware changed.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE

Reply:
thanks, I will take a look, maybe we can return body instead of nil to avoid this big breaking change for now. Will let you know 🙂

Reply:
I found some time to look at this again so I just pushed a change to `Connection#delete` that preserves compatibility. Happy to make any other changes as well if you have any other feedback!

Reply:
It looks like we could also modify the `FixUnparsedJson` middleware to check for `nil` to accomplish the same thing.